helper/send_json_response.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // SendJSONResponse sends a JSON response
-func SendJSONResponse(w http.ResponseWriter, status bool, statusCode int, message string, data map[string]interface{}, opts ...interface{}) {
```

Comment:
I believe `opts ...interface{}` should be added because this is a template and it will be useful for some users of this template.

Reply:
okay